controllers/apply/form-router-next/index.js

```diff
@@ -427,24 +425,22 @@ function initFormRouter({
 
     /**
      * Routes: Stream file from S3 if authorised
+     * Stream the file's headers and serve it directly as a response
+     * @see https://stackoverflow.com/a/43356401
      */
     router.route('/download/:fieldName/:filename').get((req, res, next) => {
         const { currentlyEditingId, currentApplicationData } = res.locals;
 
-        // Check that this application has data for the requested field name
         const fileData = currentApplicationData[req.params.fieldName];
+        const matchesField =
+            fileData && fileData.filename === req.params.filename;
 
-        // Confirm that the requested filename matches this field's file
-        if (fileData && fileData.filename === req.params.filename) {
```

Comment:
Do you think the comments here weren't adding very much value? I appreciate they're only mirroring what the code is doing, but for me it helps parse what's going on (eg. the intention of these checks) rather than having to read through the logic and try to remember which variables come from where etc.

Reply:
I typically find that these kind of comments are better replaced with a variable or function name with the intent of the comment. Tried to do that by assigning it to `matchesField`.